THIRD-PARTY-NOTICES

```diff
@@ -86,3 +86,48 @@ https://github.com/mongodb/mongo-csharp-driver.
      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
      See the License for the specific language governing permissions and      
      limitations under the License.                                             
+
+3. The following files: NativeBufferInfo.cs, Zstandard64NativeMethods.cs, ZstandardNativeWrapper.cs, ZstandardStream.cs
```

Comment:
Original license:
https://github.com/bp74/Zstandard.Net/blob/master/LICENSE

I took this as an example: https://github.com/vincentkam/mongo-csharp-driver/blob/master/THIRD-PARTY-NOTICES#L15-L88